_includes/head.html

```diff
@@ -17,5 +17,5 @@
 
 <!-- For all browsers -->
 <link rel="stylesheet" href="{{ base_path }}/assets/css/main.css">
-
-<meta http-equiv="cleartype" content="on">
\ No newline at end of file
+<link rel="stylesheet" href="{{ base_path }}/assets/css/primer.css"
```

Comment:
@hectorsector is it http://www.rubydoc.info/gems/jekyll-theme-primer/0.1.6 you were looking at?